adapters/handlers/grpc/v1/service.go

```diff
@@ -49,6 +49,39 @@ func NewService(traverser *traverser.Traverser, authComposer composer.TokenFunc,
 	}
 }
 
+func (s *Service) BatchDelete(ctx context.Context, req *pb.BatchDeleteRequest) (*pb.BatchDeleteReply, error) {
+	before := time.Now()
+	principal, err := s.principalFromContext(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("extract auth: %w", err)
+	}
+	replicationProperties := extractReplicationProperties(req.ConsistencyLevel)
+	scheme := s.schemaManager.GetSchemaSkipAuth()
+
+	params, err := batchDeleteParamsFromProto(req, scheme)
+	if err != nil {
+		return nil, err
```

Comment:
Can we wrap this error (and other errors in this function) to show the location in which they were thrown? E.g. `fmt.Errorf("batch delete: %w", err)` and likewise for other errors in this PR

Actually, I feel this should be the same for all the methods in the gRPC API.
Perhaps we can do them all in another PR?

Reply:
I added it in the service file, but don't want to add it in all subfunctions - that is just too much copy&paste and often not really needed